packages/flutter/test/gestures/long_press_test.dart

```diff
@@ -9,153 +9,275 @@ import 'gesture_tester.dart';
 
 const PointerDownEvent down = PointerDownEvent(
   pointer: 5,
-  position: Offset(10.0, 10.0)
+  position: Offset(10, 10),
 );
 
 const PointerUpEvent up = PointerUpEvent(
   pointer: 5,
-  position: Offset(11.0, 9.0)
+  position: Offset(11, 9),
+);
+
+const PointerMoveEvent move = PointerMoveEvent(
+  pointer: 5,
+  position: Offset(100, 200),
 );
 
 void main() {
   setUp(ensureGestureBinding);
 
-  testGesture('Should recognize long press', (GestureTester tester) {
-    final LongPressGestureRecognizer longPress = LongPressGestureRecognizer();
-
-    bool longPressRecognized = false;
-    longPress.onLongPress = () {
-      longPressRecognized = true;
-    };
-
-    longPress.addPointer(down);
-    tester.closeArena(5);
-    expect(longPressRecognized, isFalse);
-    tester.route(down);
-    expect(longPressRecognized, isFalse);
-    tester.async.elapse(const Duration(milliseconds: 300));
-    expect(longPressRecognized, isFalse);
-    tester.async.elapse(const Duration(milliseconds: 700));
-    expect(longPressRecognized, isTrue);
-
-    longPress.dispose();
-  });
+  group('Long press', () {
```

Comment:
I'm assuming that none of the tests in this file actually changed?

Reply:
The existing ones no. They were just moved into the group. Some additional ones were added.